src/strategies/cyberkongz-v3/index.ts

```diff
@@ -0,0 +1,71 @@
+import { formatUnits } from '@ethersproject/units';
+import {getProvider, getSnapshots, multicall} from '../../utils';
+
+export const author = 'maxbrand99';
+export const version = '1.0.0';
+
+const bananaContract = '0xe2311ae37502105b442bbef831e9b53c5d2e9b3b';
+
+const abi = [
+  'function balanceOf(address account) external view returns (uint256)',
+  'function totalSupply() external view returns (uint256)'
+];
+
+export async function strategy(space, network, provider, addresses, options, snapshot) {
+  const allAddresses = {};
+  const promises: any = [];
+  const blocks = await getSnapshots(network, snapshot, provider, options.chains.map((s) => s.network || network));
+  const allCalls: any[] = [];
+  options.chains.forEach((chain) => {
+    const calls: any[] = [];
+    Object.keys(chain.registries).forEach((registry) => {
+      allAddresses[registry] = chain.registries[registry]
+      addresses.forEach((address: any) => {
+        calls.push([registry, 'balanceOf', [address]]);
+        allCalls.push([registry, 'balanceOf', [address]]);
+      });
+    });
+    const blockTag = typeof blocks[chain.network] === 'number' ? blocks[chain.network] : 'latest';
+    promises.push(
+      multicall(chain.network, getProvider(chain.network), abi, calls, {blockTag})
+    );
+  })
+
+  const results = await Promise.all(promises);
```

Comment:
And if so it will cross our 5 call limit

Reply:
wouldn't the exact same thing happen here?
https://github.com/snapshot-labs/snapshot-strategies/tree/master/src/strategies/multichain
I could request as many chains as i want, easily passing the limit of 5 calls

Reply:
Yes yes multichain is an exception, we wanted to apply a limit for it too, but don't want to break existing settings (It doesn't have any limit because this strategy was created long back before our core multichain feature and many spaces still depend on it 😅 )